src/sidebar/components/thread-list.js

```diff
@@ -46,16 +51,20 @@ function getScrollContainer() {
 function ThreadList({ thread }) {
   // Height of the visible area of the scroll container.
   const [scrollContainerHeight, setScrollContainerHeight] = useState(
-    getScrollContainer().clientHeight
+    window.innerHeight
```

Comment:
`window.innerHeight` is just used as an initial default here which doesn't depend on anything rendered by a parent component.